src/createServer.js

```diff
@@ -1,3 +1,35 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { validateError } = require('./validateError');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+  return http.createServer((req, res) => {
+    const normilizedURL = new URL(req.url, `http://${req.headers.host}`);
+    const originalText = normilizedURL.pathname.slice(1);
+    const targetCase = normilizedURL.searchParams.get('toCase');
+
+    res.setHeader('Content-Type', 'application/json');
+
+    const errors = validateError(originalText, targetCase);
```

Comment:
You don't validate errors, but incoming arguments. The part of the name Error is incorrect here.